core/consensus/grandpa/impl/grandpa_impl.cpp

```diff
@@ -109,15 +109,17 @@ namespace kagome::consensus::grandpa {
     }
 
     current_round_ = makeInitialRound(round_state, std::move(voters));
-    BOOST_ASSERT_MSG(current_round_ != nullptr,
-                     "Initial round must create successful in any case");
-
-    GrandpaImpl::executeNextRound(current_round_->roundNumber());
+    BOOST_ASSERT(current_round_ != nullptr);
```

Comment:
Why remove the message?

Reply:
It never returns nullptr. We can remove assert too.